src/trio/_core/_run.py

```diff
@@ -2952,6 +2952,22 @@ async def checkpoint_if_cancelled() -> None:
     task._cancel_points += 1
 
 
+def in_trio_run() -> bool:
+    """Check whether we are in a Trio run.
+
+    See also :ref:`the different types of contexts <trio_contexts>`.
```

Comment:
```suggestion
    """Check whether we are in a Trio run.
    This is true if and only if :func:`trio.current_time` will succeed.

    See also the discussion of what it means to be :ref:`in "Trio context" <trio_contexts>`.
```
and then analogously for `in_trio_task`